fdbserver/fdbserver.actor.cpp

```diff
@@ -1703,7 +1703,7 @@ struct CLIOptions {
 				break;
 			}
 			case OPT_KMS_REST_ALLOW_NOT_SECURE_CONECTION: {
-				TraceEvent("RESTKmsConnAllowNotSecureConnection");
+				TraceEvent(SevWarnAlways, "RESTKmsConnAllowNotSecureConnection");
```

Comment:
Why `Warn`? I think it is more of an information message as for test/dev it is valid _not_ to have insecure connection

Reply:
I think the idea is if we accidentally enable this on production